pywbem_mock/_mainprovider.py

```diff
@@ -1040,50 +1129,53 @@ def ModifyClass(self, **params):
     #
```

Comment:
In the exception above, the class name is incorrect. Change `Fake_WBEMConnection` to `FakedWBEMConnection`.

Reply:
Done in cleanup. DONE